tests/SimpleStringTest.cpp

```diff
@@ -560,20 +560,15 @@ TEST(SimpleString, CollectionWritingToEmptyString)
     STRCMP_EQUAL("", col[3].asCharString());
 }
 
-#ifdef CPPUTEST_64BIT
-#ifdef CPPUTEST_64BIT_32BIT_LONGS
+#if !defined(CPPUTEST_64BIT) || \
+     defined(CPPUTEST_64BIT_32BIT_LONGS)
```

Comment:
I don't think so, from what I can tell the first block is for systems that don't support 64bit at all or still have a 32bit long on a 64bit platform (windows x64).  It has an empty IGNORE_TEST in it.  The #else condition has the functional test in it that check the 64-bit long behavior.